lib/cdo/shared_constants/curriculum/shared_course_constants.rb

```diff
@@ -77,6 +77,56 @@ module SharedCourseConstants
       other
     ).freeze
 
+    # The grade bands used to represent course offering audience
+    COURSE_OFFERING_GRADE_BANDS = OpenStruct.new(
+      {
+        elementary: 'Elementary',
+        middle: 'Middle',
+        high: 'High'
+      }
+    ).freeze
```

Comment:
Do we still need this?

Reply:
I thought about removing this, but realized we will end up making it anyway once we start front-end. Happy to remove until then?

Reply:
https://codedotorg.atlassian.net/browse/TEACH-272